### DISCUSSION TITLE
pkg tree view

### DISCUSSION THREAD

**Discussion by zzz6093** [microsoft/terminal#14613] [General]
windows terminal을 이용하여 wsl을 사용하고 있습니다.
tree패키지를 사용하여 경로를 tree형식으로 볼 때 '|' 문자가 잘리는 것 같습니다.
windows terminal 사용 시 ( tree command)
![image](https://user-images.githubusercontent.com/35140301/210717289-df31c8f9-51e5-4134-8291-8b3bf090655c.png)
wsl 사용 시
![image](https://user-images.githubusercontent.com/35140301/210717446-a2136d5d-17d5-4b3d-9b70-89d86d6da5ab.png)

**Comment by zadjii-msft** [comment]
Sorry, I think we're going to need more details than that to help you. If you post your question in your native language, there's a chance that we might be able to use a machine translator to help you. No promises though 😕

**Reply by zzz6093** [reply]
감사합니다 모국어 및 이미지를 추가하여 다시 작성하였습니다.

**Comment by zadjii-msft** [comment]
Thanks for elaborating! The pictures really helped. I bet this is due to whatever font you have selected. Does trying a different font cause those lines to appear? We've had issues with line-drawing characters in certain fonts, at certain sizes, in the past. See #9235, #5897

**Reply by zzz6093** [reply]
감사합니다 해당 문제는 글꼴 사이즈 조절을 하면 나타나지 않는 것 같습니다.!!
